.github/workflows/acceptance_test.yml

```diff
@@ -186,16 +186,6 @@ jobs:
         with:
           name: reports_all
           path: ${{ github.sha }}/output
-      - name: Set up and authorize Cloud
```

Comment:
Could you describe more what we lose by disabling this? Can we still download the reports as linked in the comment generated by the acceptance test (e.g., "Download the full acceptance test report for commit https://github.com/MobilityData/gtfs-validator/commit/6e94daab39c7560d996a3dc5d86b21c15aac49ce [here] (report will disappear after 90 days)"? If not does that comment need to change?

If it possible to check here if the Action has permission to use the credentials? If so, we could still run this block for branches on this repo, but just not for 3rd party forks.

Reply:
> Could you describe more what we lose by disabling this?
By doing so, we loose the ability to store validation reports on GCP Cloud storage. But it is still possible to download them from the internal storage of github. 

> Can we still download the reports as linked in the comment generated by the acceptance test (e.g., "Download the full acceptance test report for commit https://github.com/MobilityData/gtfs-validator/commit/6e94daab39c7560d996a3dc5d86b21c15aac49ce [here] (report will disappear after 90 days)"? If not does that comment need to change?

Yes - this remains doable. 

> If it possible to check here if the Action has permission to use the credentials? If so, we could still run this block for branches on this repo, but just not for 3rd party forks.
 
Let me see.

Reply:
From a [quick search](https://github.community/t/how-can-i-test-if-secrets-are-available-in-an-action/17911) it appears that checking the presence of a repo secret in a `if` block of a job's step might not be possible. However, it might be possible to include an intermediate step that consists in checking the presence of said secret and store the boolean in an environmental variable, then execute `Set up and authorize Cloud` and `Upload reports to Google Cloud Storage` conditionally. Unfortunately I will not have the time to do it 😔.

Reply:
Looks like this is the issue regarding making execution conditional on secrets - https://github.com/actions/runner/issues/520